spec/support/webmock.rb

```diff
@@ -106,21 +106,23 @@
     ).to_return(
       status: 400,
       headers: { 'Content-Type' => 'application/vnd.api+json' },
-      body: { defendant_id: ['is not a valid uuid'],
-              maat_reference: [
-                '1234567 has no common platform data created against Maat application.'
-              ] }.to_json
+      body: {
+        'error' =>
+          'Contract failed with: {:defendant_id=>[\"is not a valid uuid\"]},
```

Comment:
CDA will return `422` for contract failures.

Reply:
👍 have corrected